### PR TITLE
Armando/grwth 2188 two og images conflict on v2 marketplace

### PR DESCRIPTION
**Pull request by r3lays**

### CODE REVIEW

File: components/Head.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { FC } from 'react'
+import NextHead from 'next/head'
+
+type Props = {
```
Comment (r3lays) [inline components/Head.tsx:4]: These can be expanded as needed. They were the only values that were set throughout the marketplace.

**Comment by r3lays** [comment]
Looked over the design with Lingzhi and she cleared it.

File: components/Head.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  ogImage = 'https://marketplace.reservoir.tools/og-image.png',
+  title = 'Reservoir Market | Open Source NFT Marketplace',
+  description = 'Reservoir Market is an open source NFT marketplace built with Reservoir.',
+}) => {
+  return (
+    <NextHead>
+      {/* CONFIGURABLE: You'll probably want to configure this all to have custom meta tags and title to fit your application */}
+
+      {/* Meta tags */}
+      <meta name="keywords" content="nft, ethereum, protocol" />
```
Comment (pedromcunha) [inline components/Head.tsx:26]: Is it possible to just have the tags that are dynamic in here? E.g. keep the keywords, reservoir tags and anything else that's relatively static in the document page?

**Review by pedromcunha — CHANGES_REQUESTED** [review]

File: components/Head.tsx (hunk 2 above)
Comment (r3lays) [reply components/Head.tsx:26]: Yes! I just didn't want to separate them because I wanted all the tags to be in the same spot.

I'll move them right now.

Comment (r3lays) [reply components/Head.tsx]: Moved more tags to the global scope _document

File: pages/_document.tsx [hunk 3]
```diff
@@ -25,49 +22,28 @@ class MyDocument extends Document {
...
-        <meta property="og:locale" content="en" />
-        <meta property="og:description" content={description} />
-        {/* The optimal size is 1200 x 630 (1.91:1 ratio). */}
-        <meta property="og:image" content={ogImage} />
-        <meta property="og:image:type" content="image/png" />
-        <meta property="og:image:width" content="1200" />
-        <meta property="og:image:height" content="630" />
-        <meta property="og:image:alt" content="Reservoir Market Banner" />
-
-        {/* CONFIGURABLE: Reservoir meta tags can be configured to your application, learn more here: https://docs.reservoir.tools/docs/reservoir-meta-tags */}
```
Comment (pedromcunha) [inline pages/_document.tsx:68]: Missing this CONFIGURABLE tag

**Review by pedromcunha — CHANGES_REQUESTED** [review]

**Pull request merged**